scripts/build/release_publisher/publisher_test.go

```diff
@@ -72,10 +75,12 @@ func TestPreparingReleaseFromRemote(t *testing.T) {
 			expectedBeta:    false,
 			expectedStable:  false,
 			expectedArch:    "armv6",
-			expectedOs:      "linux",
-			expectedURL:     "https://s3-us-west-2.amazonaws.com/grafana-releases/release/grafana-rpi-5.4.0-pre1asdf_armhf.deb",
+			expectedOs:      "deb",
```

Comment:
Don't understand this change. Was it a "test mistake" using linux earlier?

Reply:
@marefr Well noticed, but yeah I do think it must've been a mistake in the test before. If you look at the corresponding [production configuration](https://github.com/grafana/grafana/blob/master/scripts/build/release_publisher/publisher.go#L132), the OS is "deb" (which in this codebase corresponds to the package type).